scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -0,0 +1,498 @@
+##! Module for recording and querying metrics. This modules wraps
+##! the lower-level telemetry.bif functions.
+
+module Telemetry;
+
+export {
+	type labels_table: table[string] of string;
+	type labels_vector: vector of string;
+
+	## Type that captures options used to create metrics.
+	type MetricOpts: record {
+		## The prefix (namespace) of the metric.
+		prefix: string;
+
+		## The human-readable name of the metric.
+		name: string;
+
+		## The unit of the metric. Use the pseudo-unit "1" if this is a unit-less metric.
+		unit: string;
+
+		## Documentation for this metric.
+		helptext: string &default="Zeek Script Metric";
```

Comment:
```suggestion
		help_text: string &default="Zeek Script Metric";
```

Or maybe just `doc`?

Reply:
... and I think I'd leave this optional instead of a having a default (because the default doesn't seem very helpful on its own).

Reply:
The bifs use  `helptext`, but those are positional and better to have `_` it here.

> Or maybe just doc?
I'll stick with "help" due to the C++ API and broker using that throughout.

Hmm, I'll go for making it required (ping if you object). There's some prom tooling that will point out missing help text and it's not too cumbersome to add it and provides documentation, too. It' not necessarily saving on the verbosity, too.